numpy/_core/src/umath/scalarmath.c.src

```diff
@@ -234,7 +234,7 @@ static inline int
  */
 
 static inline int
-@name@_ctype_true_divide(npy_@name@ a, npy_@name@ b, npy_double *out)
+@name@_ctype_true_divide(npy_double a, npy_double b, npy_double *out)
 {
     *out = (npy_double)a / (npy_double)b;
```

Comment:
Now don't have to cast any more.

More generally, why is the `@name@` still needed?

Reply:
Or does this all work fine, and the cast to `npy_double` is now implied by calling the function? I'm confused why this does not give compiler errors.